scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/dao/RecurringScheduleDaoImplTest.java

```diff
@@ -55,13 +59,16 @@ public void before() throws SQLException {
 		// Add fake test records.
 		String appId = "appId1";
 		String guid = TestDataSetupHelper.generateGuid();
-		List<RecurringScheduleEntity> entities = TestDataSetupHelper.generateRecurringScheduleEntities(appId, guid, 1, 0);
+		List<RecurringScheduleEntity> entities = TestDataSetupHelper.generateRecurringScheduleEntities(appId, guid,false, 1,
```

Comment:
You can use RecurringScheduleEntitiesBuilder  class to create entities instead of TestDataSetupHelper class.
entities = RecurringScheduleEntitiesBuilder builder = new RecurringScheduleEntitiesBuilder(1, 0).setAppId(appId).setGuid(guid).setTimeZone(timeZone).setDefaultInstanceMinCount(1).setDefaultInstanceMaxCount(5).build();

Because we plan to get rid of TestDataSetupHelper later.